data/mods/Aftershock/items/armor/winter_masks.json

```diff
@@ -0,0 +1,128 @@
+[
+  {
+    "id": "afs_magellan_suit_helmet",
+    "repairs_like": "afs_magellan_suit",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "looks_like": "helmet_motor",
+    "name": { "str": "Magellan helmet CA." },
+    "description": "The high quality helmet of a Magellan exosuit, adapted to handle the freezing but breathable air of Salus IV.  In addition to its life support functionality, it features a minor augmented reality UI overlay and a retractable gold-plated visor to protect against glare and UV light.  Although not armored as such, it's strong enough to handle minor blunt impacts.",
+    "weight": "2500 g",
+    "volume": "2250 ml",
+    "price": "750 USD",
+    "to_hit": -1,
+    "bashing": 7,
+    "material": [ "plastic", "nomex" ],
+    "symbol": "[",
+    "color": "dark_gray",
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": {
+      "type": "transform",
+      "msg": "You activate your %s.",
+      "target": "afs_magellan_suit_helmet_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The %s's batteries are dead."
+    },
+    "armor_portion_data": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 25 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 5 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 15 }
+    ],
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "battery": 1000 } } ],
+    "warmth": 10,
+    "material_thickness": 6,
+    "environmental_protection": 1,
+    "techniques": [ "WBLOCK_1" ],
+    "flags": [
+      "VARSIZE",
+      "WATERPROOF",
+      "GAS_PROOF",
```

Comment:
Should it be "GAS_PROOF" when uncharged?  Would we want to include a note that it needs a filter when multimag becomes a reality, if left as is?

Reply:
Ah probably not.